### DISCUSSION TITLE
Voting and coalition system

### DISCUSSION THREAD

**Discussion by osAIran** [tcfev/forDem#160] [Ideas]
Hey everyone.

For digital democracy we need to have an excellent voting platform that has the following:

- Direct voting on laws/plans/goals
- Explanation/Education of laws/plans/goals in plain native languages with examples and arguments for and against them. Present expert opinions and debates on them 
- Proposing plans/laws/goals
- Open discussions on proposed plans/laws/goals (multimedia/streaming/text/etc)
- Present statistical analysis of voting (e.g., group by age)
- Fork and propose an upgrade of law/plan/goal 
- Categorization and separation of concerns (e.g., law, digital laws, education, health care, etc.)
- Different levels and layers of voting (On overall ideas, more specific (plans) and actual law) 

Some Ideas that we need to explore later:
- Fluid voting
- Counting votes (e.g., in a law on abortion, women's vote should for example be counted as double)
- Decide how to count minor votes (e.g., limit to the overall idea? use fluid democracy so that the parent decided unless the child decides/can vote?)

To bootstrap digital democracy, we need to have a demo ASAP to showcase to the public and present a platform to form coalitions on plans/laws/goals for the future. 

Let me know what you think. We need to improve this as much as possible.

**Comment by armantorkzaban** [comment]
Legit. 
It is related to the following issues: 
#24 #10 #141 #15 #161 
You also mentioned a few items which could be defined as new features: 

- [ ] positive discrimination/ add weight to vote of certain voters
- [ ] fork/variations of same law/plan/goal/suggestion

To pull this off timely we need more contribution from the community.

**Reply by osAIran** [reply]
Ok. I think we should make a GitHub account with the name of iran digital democracy to connect to more people. This project is unfortunately not well known and we need to connect to as much as we can

**Reply by desputinski** [reply]
You mean as a fork of forDem? Or an identical app by source code that just has a different name? If it helps, why not!

**Reply by osAIran** [reply]
yeah but even with another name unless we have a good PE campaign the problem still remains
